packages/vanilla/src/components/horizontalrule/README.md

```diff
@@ -0,0 +1,91 @@
+# Horizontal Rule
+
+> The horizontal rule component is to be utilized within IBM.com for thematic
+> breaks within the content of the page.
+
+## Getting started
+
+Here's a quick example to get you started:
+
+### ES6 Template Literal
```

Comment:
Should we add an area for an HTML example?

````markdown
## Getting started

Here's a quick example to get you started:

### HTML
Copy and paste the following into your HTML `body` to get started.

```html
<hr data-autoid="dds--hr" class="bx--hr" />
```

### ES6 Template Literal
````

Reply:
It's actually below the `ES6 Template Literal` section. Should we bring that above it? I'm fine with either way, but at least both options are already in the README.

Reply:
I took that as what's returning from the ES6 Template Literal console. It might be good to have a specific place called out where the user can copy it from more clearly defined. Just a suggestion though.